api/src/main/java/io/opentelemetry/metrics/LongSumObserver.java

```diff
@@ -16,33 +16,42 @@
 
 package io.opentelemetry.metrics;
 
-import io.opentelemetry.metrics.LongSumObserver.ResultLongObserver;
+import io.opentelemetry.metrics.LongSumObserver.ResultLongSumObserver;
 import java.util.Map;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Observer to report pre-aggregated metrics with double value.
+ * {@code SumObserver} is the asynchronous instrument corresponding to Counter, used to capture a
+ * monotonic sum with Observe(sum).
+ *
+ * <p>"Sum" appears in the name to remind that it is used to capture sums directly. Use a
+ * SumObserver to capture any value that starts at zero and rises throughout the process lifetime
+ * and never falls.
+ *
+ * <p>A {@code SumObserver} is a good choice in situations where a measurement is expensive to
+ * compute, such that it would be wasteful to compute on every request.
  *
  * <p>Example:
  *
  * <pre>{@code
  * class YourClass {
  *
  *   private static final Meter meter = OpenTelemetry.getMeterRegistry().get("my_library_name");
- *   private static final LongSumObserver observer =
+ *   private static final LongSumObserver cpuObserver =
  *       meter.
- *           .longSumObserverBuilder("doWork_latency")
- *           .setDescription("gRPC Latency")
+ *           .longSumObserverBuilder("cpu_usage")
```

Comment:
Also here

Reply:
It is because it is the total usage time since beginning of the process.